src/FlatButton/FlatButton.js

```diff
@@ -177,6 +181,7 @@ class FlatButton extends Component {
       backgroundColor,
       children,
       disabled,
+      disableTouchRipple,
```

Comment:
No need to explicit the property, it's going to be applied on the right element.